runtime/POSIX/fd.h

```diff
@@ -93,7 +98,8 @@ extern exe_sym_env_t __exe_env;
 
 void klee_init_fds(unsigned n_files, unsigned file_length,
                    unsigned stdin_length, int sym_stdout_flag,
-                   int do_all_writes_flag, unsigned max_failures);
+                   int do_all_writes_flag, unsigned max_failures,
+				   char* sym_pmem_filename, unsigned sym_pmem_size);
```

Comment:
At some point, we will want to support multiple pmem files, but for now this should be fine.